src/odemis/gui/cont/acquisition/cryo_acq.py

```diff
@@ -288,6 +294,13 @@ def _acquire_at_features(self, _: wx.Event):
         # the acquisition is started
         self._tab_data.main.is_acquiring.value = True
 
+        # there is a race condition on the is_acquiring/_on_acquisition callback
+        # if the stream is playing, the callback can start, and then the acquisition starts
+        # the callback will pause the stream, and the acquisition will timeout
+        # so we pause the stream synchronously here
+        # Note: it occurs less here because of the stage movement buffer time
```

Comment:
I don't think you need to explain the bug in the comment. This explanation should be in the commit message. Here, you can say just "The acquisition manager expects all streams to be paused before starting". If you want, you can add "Note that this cannot be done by .is_acquiring callback, because it might be delayed".